pkg/repositories/transformers/execution.go

```diff
@@ -293,6 +343,22 @@ func FromExecutionModel(executionModel models.Execution) (*admin.Execution, erro
 	}, nil
 }
 
+func GetStateFromModelOldExecs(executionModel models.Execution) (*admin.ExecutionStateChangeDetails, error) {
```

Comment:
not a huge fan of this name, could we change it to something like 'PopulateDefaultStateChangeDetails` and add a comment on why it exists?

Reply:
Done